tests/api2/test_300_nfs.py

```diff
@@ -261,40 +291,230 @@ def test_29_removing_nfs_mountpoint(request):
     assert results['result'] is True, results['output']
 
 
-def test_30_delete_nfs_share(request):
-    depends(request, ["pool_04"], scope="session")
-    nfsid = GET('/sharing/nfs?comment=My Test Share').json()[0]['id']
-    results = DELETE(f"/sharing/nfs/id/{nfsid}")
+def test_30_add_second_nfs_path(request):
+    """
+    Verify that adding a second path generates second exports entry
+    Sample:
+
+    "/mnt/dozer/NFSV4/foo"\
+    	*(sec=sys,rw,subtree_check)
+    "/mnt/dozer/NFSV4/foobar"\
+    	*(sec=sys,rw,subtree_check)
+    """
+    depends(request, ["pool_04", "ssh_password"], scope="session")
+
+    paths = [NFS_PATH, NFS_PATH]
+    results = PUT(f"/sharing/nfs/id/{nfsid}/", {'paths': paths})
+    assert results.status_code == 200, results.text
+
+    exports_paths = [x['path'] for x in parse_exports()]
+    diff = set(exports_paths) ^ set(paths)
```

Comment:
The same path is exported twice, isn't it? `set` will have only one item so I don't understand how will this test ensure that we have two exports?

Reply:
Dumb oversight on my part. :)
I've updated to create two subdirectories and then add them both as paths (which is more in line with how this feature is designed anyway).